README.md

```diff
@@ -15,11 +15,14 @@ Ready to use blocks to configure popular tools like *Babel*, *PostCSS*, *Sass*,
 ## Table of Contents
 
 - [Installation](#installation)
-- [Usage](#usage)
-- [Available webpack blocks](#available-webpack-blocks)
-- [Design principles](#design-principles)
+- [Example](#example)
+- [More Examples](#more-example)
+- [Custom Blocks](#custom-blocks)
+- [Available Webpack Blocks](#available-webpack-blocks)
+- [Third-party Blocks](#third-party-blocks)
+- [Design Principles](#design-principles)
 - [FAQ](#faq)
-- [Like what you see?](#like-what-you-see)
+- [Like What You See?](#like-what-you-see)
```

Comment:
Mixed feeling. Looks a bit strange indeed, but inconsistency is annoying as well... Not sure, mixed feelings about both our options 😅

PS: Same applies for "Available Webpack Blocks" (strange to see uppercase webpack, esp. if it's not even the first word in a sentence.